utils/bot.py

```diff
@@ -25,8 +26,11 @@ def __init__(self):
             ),
             debug_guilds=config["debug_guilds"],
         )
-
+        self.http_session = ClientSession(
+            connector=TCPConnector(resolver=AsyncResolver(), family=socket.AF_INET)
+        )
         self.on_ready_fired = False
+        self._seen_messages = 0
```

Comment:
What's this for?

Reply:
to be honest, for nothing... or just to keep track?

Reply:
I think it would be the best to keep it out of this pr as it's unrelated.

Reply:
removed. well that small var could hardly make any difference